src/apy/anki.py

```diff
@@ -34,43 +40,61 @@ def __init__(self, base=None, path=None, profile=None, **_kwargs):
         self.deck_names = self.deck_name_to_id.keys()
         self.n_decks = len(self.deck_names)
 
-    def _init_load_collection(self, base, path, profile):
-        """Load the Anki collection"""
-        # Save CWD (because Anki changes it)
-        save_cwd = os.getcwd()
-
-        if path is None:
-            if base is None:
+    def _init_load_profile(self, base, path):
+        """Load the Anki profile from database"""
+        if base is None:
+            if path is not None:
```

Comment:
What is the difference between "base" and "path" here?

Reply:
`base` is the basepath to the Anki profile. `path` is the path to the collection database.

Reply:
I guess, as a minimum, I should make better names here!